### DISCUSSION TITLE
Welcome to mlx-vlm Discussions!

### DISCUSSION THREAD

**Discussion by Blaizzy** [Blaizzy/mlx-vlm#130] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avishekjana** [comment]
Hi,
I’m an AI & ML engineer, and I see great potential in MLX for fine-tuning workflows. Could you please provide proper documentation (at least for the supported models)? It would make adoption and experimentation much smoother.
Thanks!
